src/Components/RegAssistCodeBlock/RegAssistCodeBlock.js

```diff
@@ -7,7 +7,7 @@ import {
   ClipboardCopyButton,
 } from '@patternfly/react-core';
 
-const RegAssistCodeBlock = ({ code = [] }) => {
+const RegAssistCodeBlock = ({ code = [], classname }) => {
```

Comment:
Why are we passing in this classname prop?

Reply:
because I'm adding a classname for the RHEL9 content to add the margin

Reply:
woops, I only saw the classname on the first instance not the second. a nitpick would be just to change the name to something like pfclass or attribute but not neccessary